packages/cfpb-atomic-component/src/utilities/transition/MaxHeightTransition.js

```diff
@@ -42,6 +45,15 @@ function MaxHeightTransition( element ) {
     return this;
   }
 
+  /**
+   * The whole page has loaded,
+   * including all dependent resources such as stylesheets and images.
+   */
+  function _pageLoaded() {
+    window.removeEventListener( 'load', _pageLoaded );
```

Comment:
This would be a good use case for the [`once` parameter](https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#parameters) but it looks like IE doesn't support it. :(

Reply:
TIL